File: Controller/Modelo/Ticket.cs
namespace Intermodular_MVC_VladimirIriarte.Modelo
{
    public class Ticket
    {
        public int codigo { get; set; }
        public decimal precio_total { get; set; }
        public string comentario { get; set; }

        public Ticket()
        {
        }

        public Ticket(int codigo, decimal precio_total, string comentario)
        {
            this.codigo = codigo;
            this.precio_total = precio_total;
            this.comentario = comentario;
        }

        public Ticket(decimal precio_total, string comentario)
        {
            this.precio_total = precio_total;
            this.comentario = comentario;
        }
    }
}

File: Model/controllers/ticket.js
var conexion = require('./conexion');

exports.list = function (req, res) {
    conexion.query("SELECT * FROM ticket", (error, rows) => {
        if (error) {
            console.log(error);

        } else {
            return res.json(rows);
        }
    });

}

exports.get = function (req, res) {
    var codigo = req.params.codigo;
    conexion.query("SELECT * FROM ticket WHERE codigo = ?", [codigo], (error, rows) => {
        if (error) {
            console.log(error);
        } else {
            return res.json(rows);
        }
    });

}

exports.add = function (req, res) {
    var json = JSON.parse(JSON.stringify(req.body));

    conexion.query("INSERT INTO ticket set ?", json, (error, rows) => {
        if (error) {
            console.log(error);
        } else {
            console.log("Ticket insertado", json);
        }
    });
    res.end();
}

exports.update = function (req, res) {
    var json = JSON.parse(JSON.stringify(req.body));

    var datos = {
        codigo: json.codigo,
        precio_total: json.precio_total,
        comentario: json.comentario
    };

    conexion.query("UPDATE ticket set ? WHERE codigo = ?", [datos, json.codigo], (error, rows) => {
        if (error) {
            console.log(error);
        } else {
            console.log("Ticket actualizado: ", datos);
        }
    });
    res.end();
}

exports.delete = function (req, res) {
    var codigo = req.params.codigo;

    conexion.query("DELETE FROM ticket WHERE codigo = ?", [codigo], (error, rows) => {
        if (error) {
            console.log(error);
        } else {
            console.log("Ticket borrado: ", codigo);
        }
    });
    res.end();
}